benchmarks/NonStiffBVP/Project.toml

```diff
@@ -1,4 +1,5 @@
 [deps]
+BVProblemLibrary = "ded0fc24-dfea-4565-b1d9-79c027d14d84"
```

Comment:
These changes in Project.toml and Manifest.toml are unnecessary.

Reply:
I think BVProblemLibrary is needed, right? But yeah `Weave` should not be added.

Reply:
Oh wait this is duplicated. Erik is right, this needs to be removed